unpack_tar_file.py

```diff
@@ -0,0 +1,27 @@
+import os
+from tarfile import TarFile
+
+projects_dir = 'projects'
+
+
+def main():
+    data_path = os.getcwd() + '/' + projects_dir
+    projects = os.listdir(data_path)
+    for project in projects:
+        geo_path = data_path + '/' + project + '/geo'
+        geo_files = os.listdir(geo_path)
+        for geo_file in geo_files:
+            ext = '.tar'
+            dest_dir = geo_file[:-len(ext)]
+            dest_path = geo_path + '/' + dest_dir
+            if geo_file.endswith('_RAW' + ext) and dest_dir not in geo_files:
+                with TarFile.open(dest_path+ext) as tar_file:
+                    os.mkdir(dest_path)
+                    tar_file.extractall(dest_path)
+                print('Expanding: %s',dest_path)
```

Comment:
Shouldn't this go before the actual extraction.

```suggestion
                print('Expanding %s', dest_path)
```